src/NuGetForUnity.Tests/Assets/Tests/Editor/NuGetTests.cs

```diff
@@ -568,4 +569,63 @@ public void TestSerializeNugetPackageIdentifier(string version)
             Assert.That(deserialized.InRange(identifier), Is.True);
         }
     }
+
+    [Test]
+    [TestCase("jQuery", "3.7.0")]
+    public void TestPostprocessInstall(string packageId, string packageVersion)
+    {
+        var package = new NugetPackageIdentifier(packageId, packageVersion) { IsManuallyInstalled = true };
+        var filepath = NugetHelper.PackagesConfigFilePath;
+
+        var packagesConfigFile = new PackagesConfigFile();
+        packagesConfigFile.AddPackage(package);
+        packagesConfigFile.Save(filepath);
+
+        Assert.IsFalse(NugetHelper.IsInstalled(package), "The package IS installed: {0} {1}", package.Id, package.Version);
+
+        AssetDatabase.Refresh();
```

Comment:
```suggestion
        NugetHelper.ReloadPackagesConfig();
        AssetDatabase.Refresh();
```

Reply:
We do not want to call ReloadPackagesConfig() here since we want to test if AssetDatabase.Refresh properly does it by calling OnPostProcessAllAssets(), right?

Reply:
Oh you are right. But why hare the tests than failing.

Reply:
Not sure why they're failing on linux. I ran them locally on windows, Unity 2018.4.30f1, and they all passed...also the previous versions of this PR have been failing on the same step but not by failing tests, but by timing out if I remember correctly (after 360 minutes or so)